uspace/app/bdsh/exec.c

```diff
@@ -73,26 +70,27 @@ static int try_access(const char *f)
  */
 static char *find_command(char *cmd)
 {
-	size_t i;
-
-	found = (char *)malloc(PATH_MAX);
-
 	/* The user has specified a full or relative path, just give it back. */
-	if (-1 != try_access(cmd)) {
-		return (char *) cmd;
+	if (is_path(cmd)) {
+		if (-1 != try_access(cmd)) {
+			return str_dup(cmd);
+		}
 	}
 
+	char *found = (char *)malloc(PATH_MAX);
```

Comment:
Again, found can be NULL. Also the case of original code, but this should be fixed.